index.html

```diff
@@ -10,20 +10,20 @@
     <link rel="stylesheet" href="css/fontawesome-all.min.css">
     <link rel="stylesheet" href="css/styles.css">
 </head>
-<body>
+<body data-spy="scroll" data-target="navigation">
```

Comment:
data-target="navigation"

**missing #, needs to be you will notice your nav bar item will highlight which section you are on. this is [scrollspy](https://getbootstrap.com/docs/3.3/javascript/#scrollspy).**
data-target="#navigation"